### PR TITLE
add firewalld and ntp sync checks

### PR DESCRIPTION
**Pull request by manojsundaram**
firewalld and ntp sync checks

### CODE REVIEW

File: lib/1 [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+    _check_service_is_running 'System' 'ntpd'
+    
+
+  fi
+
+  check_network
+  check_java
+  check_database
+  check_jdbc_connector
+}
```
Comment (jamestyj) [inline lib/1]: The above file should be excluded?

File: lib/checks.sh [hunk 2]
```diff
@@ -222,16 +232,31 @@ function is_centos_rhel_7() {
...
 
   if is_centos_rhel_7; then
-    _check_service_is_not_running 'System' 'ntpd'
-    _check_service_is_running     'System' 'chronyd'
+    ntpd_used="$(_validate_service_state 'System' 'ntpd')"
+    if [ $ntpd_used -eq 0 ]; then
+      _check_service_is_running 'System' 'ntpd'
+      is_ntp_in_sync
+    else
+    # Add check to see if chrony is actually synchronizing the clock. Use the command "chronyc tracking"
```
Comment (jamestyj) [inline lib/checks.sh]: The above line should be indented by 2 spaces.

**Comment by jamestyj** [comment]
Also update your local Git configuration so that the author name and email address in the commits are correct.

**Comment by jamestyj** [comment]
And your commit message should reference issue #6.

File: lib/1 (hunk 1 above)
Comment (manojsundaram) [inline lib/1]: removed. it was a typo which automatically wrote this file out.

File: lib/checks.sh (hunk 2 above)
Comment (manojsundaram) [inline lib/checks.sh]: i dont see a space issue anywhere from the code block?

**Comment by manojsundaram** [comment]
Closing this request. Sending another with the changes discussed

**Comment by jamestyj** [comment]
In future you can make changes directly by rebasing the branch, instead of
making a new pull request.

**Pull request closed without merging**